src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -4427,6 +4427,27 @@ def ONNXRandomNormalOp:ONNX_Op<"RandomNormal",
     static std::vector<int> getTypeMap() {
       return {-1};
     }
+    std::vector<mlir::Type> resultTypeInference() {
+      std::vector<mlir::Type> resultTypes;
+      if (auto attr = dtypeAttr()) {
```

Comment:
two nits (sorry I didn't see them when I made the earlier comments):

1. I think it's clearer to say `if (dtype().has_value())` because dtype() returns Optional<int64_t> and dtypeAttr() returns an IntegerAttr, and it's a little bit confusing to introduce the attr variable when it's not used

2. lines 4433-4448 should be indented (it's curious clang-format didn't catch this)

Reply:
> lines 4433-4448 should be indented (it's curious clang-format didn't catch this)

clang-format does not scan `.inc` files I guess.

Reply:
good point

please ignore my comment about indentation, it doesn't matter in generated code

I should have commented on the generating code in gen_onnx_mlir.py (where my 1st nit still applies)